fortifyapi/client.py

```diff
@@ -584,21 +584,34 @@ def get(self, id):
         with self._api as api:
             return Issue(self._api, api.get(f"/api/v1/projectVersions/{self.parent['id']}/issues/{id}")['data'], self.parent)
 
-    def assign(self, user):
+    def assign(self, id, user, revision=0):
         """
-        :param user: user id?
+        :param id: Integer of the issue, typically a 3 to 7 digit int NOT the IID
+        :param user: user login id
+        :param revision: Begins with 0 and incremented with each assignment
         """
         self.assert_is_instance()
-        o = dict(user=user)
-        f"/api/v1/projectVersions/{self.parent['id']}/issues/action/assignUser"
+        o = {
+            "issues": [
+                {
+                    "id": id,
+                    "revision": revision
+                }
+            ],
+            "user": user
+        }
+        with self._api as api:
+            return api.post(f"/api/v1/projectVersions/{self.parent['id']}/issues/action/assignUser", o)
 
-    def audit(self,  analysis, comment="via automation", user=None, suppressed=False, tags=None):
+    def audit(self, id, analysis, user, comment="via automation", suppressed=False, tags=None, revision=0):
```

Comment:
Same comment as above, we already know the issue id, we don't need to supply it in the method

Reply:
same response as above.  how do we know which id from a list of many?